### ISSUE TITLE
multiserver: Push message service for async multiworld

### ISSUE THREAD

**Issue by Radagast81** [ArchipelagoMW/Archipelago#3487]
### What feature would you like to see?

Would be nice if there was some push message service for the big async multiworld so it becomes less of a burdon to check for new possible checks that can be made.
Would like to have a service where i can sign in and get notifications if i recieve progressive items or hints requiring items from my worlds.

**Comment by kimimaru4000** [comment]
I agree, this would be a cool feature! Implementation-wise, something with UnifiedPush like [ntfy](https://ntfy.sh) can be used to set up this infrastructure on the server side.

**Comment by imcompiling** [comment]
What about making a per-game webhook that you could set a discord channel to listen to?
